source/includes/apps/general/_configuration.md

```diff
@@ -25,34 +21,148 @@ There are 4 main folders in a Booqable app:
 ### `/config/meta.json`
 
-[JSON Schema 0.1.0](/schemas/app-0-1-0.json)
+This is the main configuration file for the app. It defines the app's metadata, capabilities, and plans. It is required for every app and is validated against a JSON schema.
 
-TODO
+Latest: [0.1.0 JSON Schema](/schemas/app-0-1-0.json) / [0.1.0 Reference](#reference)
 
-* link to reference
 
 #### Capabilities
 
-TODO brief summary with links to the relevant sections in the docs
+In your app's `meta.json` file, you can define the capabilities of your app. These are the main tasks your app can perform inside Booqable, such as:
+
+* **Tracking scripts:** Let users add analytics scripts to their website to track user behavior.
+* **Theme blocks:** Allow users to install new content blocks in their website.
+* **Delivery carriers:** Define new carriers that can be used to ship orders.
+* **Payment options:** Define new payment methods that can be used to pay for orders.
+
+For more information see the [Capabilities section](/apps.html#capabilities).
 
 
 
 ### `/config/plans.json`
 
+This file is where you define the plans your app offers. Each plan has a title, description, and a list of features presented to the user when they are installing the app.
+
 TODO
 
 
 
 ### Locale files
 
+Locale files contain the localized strings used throughout the Booqable UI such as the app title and descriptions as well forms for user settings and post-installation guides.
+
+They must be in YAML format and are stored in the `locales` folder. The file name must be the locale language code, such as `en.yaml` or `pt-BR.yaml`. The top-level of the file should contain only a single key which matches the locale's code, so `en` or `pt_BR`.
+
+
+#### App metadata
+
+```yaml
+en:
+  title: "Gizmo App"
+  description:
+    short: "A simple app to manage your gizmos."
+    long: "This app allows you to manage your gizmos. You can add, edit, and delete gizmos. Delight your customers with new gizmos every month!"
+```
+
+The app's title and descriptions are configurable through the locale files. The short description is mainly used in the app store listings page, while the long description is used in the app store detail page.
+
+
+#### App Plans
+
+```yaml
+en:
+  plans:
+    free:
+      title: Gizmo Free
+      description: A free plan to get you started with the Gizmo App.
+      features:
+      - 10 gizmos
+      - 100 notifications per month
+      - 100 API requests per month
+    pro:
+      title: Gizmo Pro
+      description: 10x the fun with the Gizmo Pro plan.
+      features:
+      - 100 gizmos
+      - 1000 notifications per month
+      - 1000 API requests per month
+```
+
+The title, description and features of each plan are configurable through the locale files. These are shown to users in the app's store detail page and as part of the [installation flow](#how-apps-work-app-store). Each plan must have a unique key under the `<locale>.plans` key, such as `free`, `basic`, `pro`, etc.
+
```

Comment:
"When a plan is no longer available, you should keep the translation keys for existing customers."